draft-irtf-cfrg-vdaf.md

```diff
@@ -3820,7 +3820,7 @@ All gadgets are listed in {{gadgets}}.
 
 | Parameter         | Value                                      |
 |:------------------|:-------------------------------------------|
-| `Valid`           | `Count(Field64)` (this section)            |
+| `Valid`           | `Count(field)` (this section)              |
 | `Field`           | `Field64` ({{fields}})                     |
```

Comment:
Let's move the `field` row to the first row of the table, as we did for `xof`, `idpf`, and `flp` elsewhere. Here and for all other variants below.